test/framework/src/main/java/org/elasticsearch/test/rest/ESRestTestCase.java

```diff
@@ -600,6 +614,10 @@ protected static void wipeAllIndices() throws IOException {
         }
     }
 
+    protected static void wipeDataStreams() throws IOException {
+        adminClient().performRequest(new Request("DELETE", "_data_stream/*"));
```

Comment:
This will likely have to catch the exception when datastreams aren't available (think BWC tests with 7.7 or earlier)

Reply:
Thanks, I'll add that